decidim-regulations/app/services/decidim/regulations/regulation_search.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Regulations
+    # Service that encapsulates all logic related to filtering participatory processes.
+    class RegulationSearch < ParticipatorySpaceSearch
+      def initialize(options = {})
+        super(ParticipatoryProcess.all, options).where(decidim_participatory_process_group_id: Rails.application.config.regulation)
+      end
+
+      def search_date
+        case date
+        when "opened"
+          query.active.where("decidim_participatory_processes.start_date < ? AND decidim_participatory_processes.end_date > date(?)", DateTime.now.to_date, DateTime.now.to_date).order(start_date: :desc)
```

Comment:
Ditto `DateTime.now` or `DateTime.current`?

Reply:
It's better `Date.current`